About.html

```diff
@@ -0,0 +1,80 @@
+        @import url('https://www.dafont.com/es/cocogoose.font');
+    </style>
+
+                
+
+</head>
+<body>
+    <section class="header">
+        <div>
+            <ul class="pre-nav-bar desktop">
+                <li class="socialMedia"><a class="icon" href="https://github.com/Kakalanp"><img src="Images/GitHub.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://www.linkedin.com/in/joaquín-garrido-lecca-zanetti-623583204/"><img src="Images/LinkedIn.png" alt=""></a></li>
+                <li class="socialMedia"><a class="icon" href="https://twitter.com/LeccaJoaquin"><img src="Images/twitter.png" alt=""></a></li>
+                <li><a class="link">English</a></li>
+                <li><a class="link">My Page</a></li>
+                <li><a class="link">Logout</a></li>
+            </ul>
+        </div>
+
+        <nav class="nav-bar">
+            <a href="index.html" class="logo desktop"><img class="logo" src="Images/Logo.png" alt="Logo"></a>
+            <ul class="menu">
+                <li><a class="active" href="About.html">About</a></li>
+                <li><a>Program</a></li>
+                <li><a>Join</a></li>
+                <li><a>Sponsors</a></li>
+                <li><a>News</a></li>
+                <li><a class="cc-campaign">CC Campaign</a></li>
+            </ul>
+            <a class="hamburger mobile" href="#">≡</a>
+        </nav>
+    </section>
+
+
+    <main>
+    <div class="about">
+        <h1>ABOUT HTML COURSE</h1>
+        <div class="spit-info-reverse"><p>The HTML course bring together a community of expert teachers who complice the educational commons required for the development of alumni as proffesional web developmers. Teaching the necessary skills to become our future web developers.</p></div>
+        <p>Pleace contact us via e-mail for any further questions regarding this HTML course</p>
+        <a class="email" href="mailto:jgarridoleccazantti@gmail.com">jgarridoleccazanetti@gmail.com</a>
+    </div>
+
+    <div class="logoabout">
+        <h2>HTML Course Logo</h2>
+        <p>The HTML Course logo was created by LogoLogo.com, a division of the DIRcreative Group, which owns a successful, brick-and-mortar graphic design company. Uploaded on <i>2014-08-03</i>, the logo is free for personal use.</p>
+        <img src="Images/Logo.png" alt="Steampunk Wings Logo">
+        <a href="https://www.logologo.com/logo.php?id=292">Download link here</a>
+        <a href="https://www.logologo.com/copyright.php">Copyright full disclosure here</a>
+    </div>
+
+    <div class="succesful-students">
+        <h2>See succesful students stories</h2>
+        <p>Take a look at the last two students who graduated from HTML Course.</p>
+        <div class="student"><a href="">Name</a><p>Student from Argentina!</p></div>
+        <div class="student"><a href="">Name</a><p>Student from Ghana!</p></div>
+    </div>
+    </main>
+    
+    <footer class="about-footer">
```

Comment:
- [ ] **Required** In the mobile version the footer should match the original website, the logo should be present on the left side & the background should be white on the mobile screen.
- Your design 
Home footer
![rwer](https://user-images.githubusercontent.com/26767383/152436106-273637a5-67cc-45e5-a382-12392a828654.PNG)
About page footer 
![xasx](https://user-images.githubusercontent.com/26767383/152436164-40c79200-519c-43d2-a7b1-e53e90aceb65.PNG)

- Original website 
![klklk](https://user-images.githubusercontent.com/26767383/152436211-552561a8-f8c5-45c4-8c50-1a145bf0d16b.PNG)